pkg/services/sqlstore/migrations/dashboard_public_mig.go

```diff
@@ -65,4 +65,13 @@ func addPublicDashboardMigration(mg *Migrator) {
 
 	// rename table
 	addTableRenameMigration(mg, "dashboard_public_config", "dashboard_public", "v2")
+	// some migrations (like AddColumn) need the table object to be passed in. So we need to make sure its updated.
+	dashboardPublicCfgV2.Name = "dashboard_public"
+
+	mg.AddMigration("add enable_annotations column", NewAddColumnMigration(dashboardPublicCfgV2, &Column{
+		Name:     "enable_annotations",
+		Type:     DB_Bool,
+		Nullable: false,
+		Default:  "0",
```

Comment:
is it ok to use "0" instead of false?

Reply:
Yeah. Bools are always represented as that anyways. The `is_enabled` column is also done the same way.